src/lib.rs

```diff
@@ -9,7 +9,7 @@
 
 #![no_std]
 
-extern crate avr_std_stub;
+#[cfg(feature = "avr-std-stub")] extern crate avr_std_stub;
```

Comment:
Why keep this at all?

Reply:
The `extern crate` statement?

Reply:
I found that `extern crate` is required - it is not enough to declare it in Cargo.toml. If the extern crate is missing, it doesn't pick up the panic handler or the eh personality from the upstream crate - it is kinda weird